teams/vsp/teams/tools/backend/external-service-monitoring/README.md

```diff
@@ -0,0 +1,43 @@
+# VSP Toolkit | SLO Tools
+
+## External Service Monitoring
+
+This folder contains tools used for managing external service monitoring reports.
+
+### Generate External Service Performance report
+
+These instructions explain how to use the `Selenium::WebDriver` based script to capture snapshots of each External Service dashboard.
+
+#### Prerequisites:
+
+These instructions and corresponding script assume the following:
+
+- `ruby --version` should be `2.3.x` or greater
+- `convert --version` should be `Version ImageMagick 6.9.x` or greater [install instructions](https://imagemagick.org/script/download.ph://imagemagick.org/script/download.php)
+- Firefox version should be `Mozilla Firefox 72.x` or greater
+- `geckodriver` [install instructions](https://github.com/mozilla/geckodriver/releases)
+
+1. Ensure your browser profile is configured to use [SOCKS proxy config](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/scripts/socks/README.md)
```

Comment:
[May also want to link to the proxy switcher docs here](https://github.com/department-of-veterans-affairs/va.gov-team/blob/f60352f90eaaed4ca4e8539cbf05a81bd519cb31/platform/engineering/internal-tools.md#chrome--firefox)